misc/package/Data Files/MWSE/core/lib/dependencyManagement/types/archives.lua

```diff
@@ -3,7 +3,6 @@ local function isArchiveActive(archive)
     local loadedArchives = tes3.getArchiveList()
     for _, loadedArchive in ipairs(loadedArchives) do
         mwse.log("archive: " .. loadedArchive)
-        ---@diagnostic disable-next-line
```

Comment:
I'm getting warnings that `loadedArchive.lower` doesn't exist

Reply:
Your usage is correct, but the annotations are wrong here. I fixed the annotations in 6709d9a49b999c580e4e145aba244c385ab0f6a7. Try rebasing this branch on master.